plf-tomcat-resources/src/main/resources/stop_eXo.sh

```diff
@@ -66,6 +66,8 @@ while [ "$1" != "" ]; do
     shift
 done
 
-export CATALINA_PID=${CATALINA_PID:-"$PRGDIR/temp/catalina.pid"}
+if [ -x "$PRGDIR/temp/catalina.pid" ]; then
```

Comment:
Do catalina.pid really have to be executable ? Shouldn't it be -w ?

Reply:
-f used instead